samples/net/sockets/socketpair/src/socketpair_example.c

```diff
@@ -144,6 +144,10 @@ int main(int argc, char *argv[])
 
 			fd = fds[i].fd;
 			idx = fd_to_idx(fd, ctx, ARRAY_SIZE(ctx));
+			if (idx < 0) {
```

Comment:
This might be 210612.  It's not 210585.

Reply:
I'll have another look soon.

Reply:
Absolutely right. Thanks for catching that.